bindings/java/src/org/sleuthkit/datamodel/Score.java

```diff
@@ -85,19 +85,19 @@
          */
 		
 		/// no significance assigned yet.
-		UNKNOWN(0, "Unknown", bundle.getString("Significance.Unknown.displayName.text")),	
+		UNKNOWN(0, "Unknown", "Significance.Unknown.displayName.text"),	
```

Comment:
Mixing work for two fixes in one PR is not ideal. In this case, the independent NPE fix is going to be held up for discussion of the OS account attributes updating changes.